### ISSUE TITLE
Add the input to be send back with the event

### ISSUE THREAD

**Issue by pevawi** [PawelDecowski/jquery-creditcardvalidator#36]
Add the input to the event so you can reference it easily from within the function enclosure for the event:

```javascript
    validate_number = function(inp, number) {
      var length_valid, luhn_valid;
      card_type = get_card_type(number);
      luhn_valid = false;
      length_valid = false;
      if (card_type != null) {
        luhn_valid = is_valid_luhn(number);
        length_valid = is_valid_length(number, card_type);
      }
      return callback(inp, {
        card_type: card_type,
        luhn_valid: luhn_valid,
        length_valid: length_valid
      });
    };
    validate = function() {
      var number;
      number = normalize($(this).val());
      return validate_number(this, number);
    };
```

**Comment by PawelDecowski** [comment]
This is a very good suggestion. However, instead of making it the callback’s argument, it can be assigned to the actual `this` variable inside the callback, by calling it using `Function.prototype.call` rather than directly.

**This is scheduled for version 1.0.**

**Comment by PawelDecowski** [comment]
This is implemented in just released [1.0](https://github.com/PawelDecowski/jquery-creditcardvalidator/blob/v1.0.0).
